### PR TITLE
Add required modifications to app.module.ts

### PR DESCRIPTION
**Pull request by rinogo**
Please feel free to tweak this as appropriate. This is a great step-by-step tutorial, but it leaves out this necessary step. Not a big deal for a NestJS expert, but a newbie (like myself) might get tripped up.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: content/recipes/prisma.md [hunk 1]
```diff
@@ -404,7 +404,17 @@ Your `UserService` and `PostService` currently wrap the CRUD queries that are av
 
 Finally, you'll use the services you created in the previous sections to implement the different routes of your app. For the purpose of this guide, you'll put all your routes into the already existing `AppController` class.
 
-Replace the contents of the `app.controller.ts` file with the following code:
+First, specify the new Prisma services in your: `app.module.ts`:
```
Comment (johannesschobel) [inline content/recipes/prisma.md:407]:
```suggestion
First, add the newly generated Prisma services to your `app.module.ts`:
```

File: content/recipes/prisma.md [hunk 2]
```diff
@@ -404,7 +404,17 @@ Your `UserService` and `PostService` currently wrap the CRUD queries that are av
...
+```typescript
+...
+@Module({
+	controllers: [AppController],
+	providers: [PrismaService, UserService, PostService],
+})
+...
```
Comment (johannesschobel) [inline content/recipes/prisma.md:414]:
```suggestion
export class ... { }
```

**Review by johannesschobel — CHANGES_REQUESTED** [review]
however, i am not sure, if these changes are really needed..

**Comment by kamilmysliwiec** [comment]
Yeah, I agree with @johannesschobel. These changes seem to be very trivial. I'd not recommend following tutorials from the "Recipes" section before going through all pages from the "Overview".

Thanks for your contribution anyways!

**Comment by rinogo** [comment]
Yes, it is a minor change, but I do think it improves the docs. Prior to running into trouble, I read the entire "Overview" as well as most of the other sections as well. However, just reading the docs doesn't mean that they're instantly and fully internalized.

I think trivial changes like this can help new users get up to speed in NestJS with less friction. Ultimately, reducing friction improves adoption - isn't that a worthy goal?

**Pull request closed without merging**